tmexp/utils.py

```diff
@@ -1,10 +1,51 @@
-from logging import Logger
+import logging
+from logging import Handler, Logger, LogRecord, NOTSET
 import os
-from typing import Counter, List, Optional
+from typing import List, Optional
 
-from .gitbase_constants import SUPPORTED_LANGUAGES
+import tqdm
 
-WordCount = Counter[str]
+SUPPORTED_LANGUAGES = [
+    "C#",
+    "C++",
+    "C",
+    "Cuda",
+    "OpenCL",
+    "Metal",
+    "Bash",
+    "Shell",
+    "Go",
+    "Java",
+    "JavaScript",
+    "JS",
+    "JSX",
+    "PHP",
+    "Python",
+    "Ruby",
+    "TypeScript",
+]
```

Comment:
BTW I just checked if we could do that with the bblfsh client. It seems we can:

    [info.name for info in client.supported_languages()]
but the list has less entries than this one:

    ['Python', 'C++', 'Java', 'JavaScript', 'Bash', 'Ruby', 'Go', 'C#', 'PHP']
which means we'd probably discard files that we can parse. @dennwc @bzz, do you know if there is a method similar to the one I use above that returns all the languages (even the “sub”-languages) that bblfsh knows how to parse?

Reply:
`client.supported_languages()` is the right way to go. 

It returns different number of languages depending on `bblfshd` runtime configuration that you are using - e.g if the typescript driver is not installed on your setup, it will not be returned. That would be the case, as default image with the drivers does include only the ones with [beta status](https://doc.bblf.sh/languages.html#supported-languages) (read: are actually maintained)

Reply:
Thanks for all the details! I did upvote the issue. We'll keep the hardcoded list while it's being addressed and will switch afterwards.

Reply:
Going to add a TODO with a link to the issue